packages/logs/src/boot/logsPublicApi.spec.ts

```diff
@@ -76,41 +79,28 @@ describe('logs entry', () => {
       setDebug(false)
     })
 
-    it('init should log an error if sampleRate is invalid', () => {
-      LOGS.init({ clientToken: 'yes', sampleRate: 'foo' as any })
-      expect(displaySpy).toHaveBeenCalledTimes(1)
-
-      LOGS.init({ clientToken: 'yes', sampleRate: 200 })
-      expect(displaySpy).toHaveBeenCalledTimes(2)
```

Comment:
we have a test covering the 'foo' case in core but it does not seem that we still have one about '200'.
what about adding it to keep this case?